src/dataframerow/utils.jl

```diff
@@ -130,13 +130,13 @@ function row_group_slots(cols::Tuple{Vararg{AbstractVector}},
             groups[i] = gix
         end
     end
-    return ngroups, rhashes, gslots, false, false
+    return ngroups, rhashes, gslots, false
 end
 
 function row_group_slots(cols::Tuple{CategoricalVector},
                          hash::Val{false},
                          groups::Union{Vector{Int}, Nothing} = nothing,
-                         skipmissing::Bool = false)::Tuple{Int, Vector{UInt}, Vector{Int}, Bool, Bool}
+                         skipmissing::Bool = false)::Tuple{Int, Vector{UInt}, Vector{Int}, Bool}
```

Comment:
maybe rename this function to `row_group_slots!` to show that it mutates its argument?

Reply:
OK